js/backend.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+(function () {
+  var TIMEOUT_IN_MS = 10000;
+  var statusCode = {
```

Comment:
statusCode с заглавной (перечисление)
```suggestion
  var StatusCode = {
```